shard.yml

```
@@ -1,10 +1,10 @@
 name: slack
-version: 0.9.0
+version: 0.9.1
 
 authors:
   - Ary Borenszweig <aborenszweig@manas.com.ar>
 
 development_dependencies:
   webmock:
     github: manastech/webmock.cr
-    version: "~> 0.9.3"
+    version: 0.12.1
```

```suggestion
    version: ~> 0.12.1
```

Better to allow 0.12.x with x >= 1